examples/grpc-service/src/main/java/example/armeria/grpc/HelloServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package example.armeria.grpc;
+
+import java.time.Duration;
+import java.util.ArrayList;
+
+import example.armeria.grpc.Hello.HelloReply;
+import example.armeria.grpc.Hello.HelloRequest;
+import example.armeria.grpc.HelloServiceGrpc.HelloServiceImplBase;
+import io.grpc.stub.StreamObserver;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class HelloServiceImpl extends HelloServiceImplBase {
+
+    /**
+     * Sends an {@link HelloReply} immediately when receiving a request.
+     */
+    @Override
+    public void hello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
+        responseObserver.onNext(buildReply(toMessage(request.getName())));
+        responseObserver.onCompleted();
+    }
+
+    /**
+     * Sends an {@link HelloReply} 3 seconds later after receiving a request.
+     */
+    @Override
+    public void lazyHello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
+        // Respond 3 seconds later.
+        Mono.delay(Duration.ofSeconds(3))
```

Comment:
I don't think it's typical for gRPC services to use reactor idioms like this and feel it makes the example hard to understand. Do you think it makes more sense to implement with armeria patterns (which also teaches RequestContext which is always very important)?

```java
RequestContext.current().contextAwareEventLoop().schedule(...)
```

Reply:
Actually we started to write this example for a user who uses gRPC and Spring WebFlux together, and we want to show how to use gRPC with Project reactor. 

But I agree with your opinion about teaching how to use `RequestContext`, so how about changing this `lazyHello` example to use `RequestContext.current().contextAwareEventLoop().schedule(...)` and leaving the `lotsOfReplies` example as it is? Make sense?